ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/map-settings/map-settings.tsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import * as React from 'react'
+import styled from '../../styles/styled-components'
+import { hot } from 'react-hot-loader'
+import Enum from '../../container/enum'
+import ExampleCoordinates from './ExampleCoordinates'
+
+type Props = {
+  selected: string
+  example: string
+  update: (selected: string) => void
+}
+
+const Root = styled<
+  { selected: string; example: string; update: (selected: string) => void },
+  'div'
+>('div')`
+  overflow: auto;
+  padding: ${props => props.theme.minimumSpacing}
+    ${props => props.theme.minimumSpacing};
+`
+
+const render = (props: Props) => {
+  const { selected, update } = props
+  return (
+    <Root {...props}>
```

Comment:
:question: Why is the same set of `props` being used on both his `Root` component and the `ExampleCoordinates` component below? Does one not need it? Does only a subset of the props apply to each component?

Reply:
Looks like the props on Root are unnecessary. I'll remove them. I'm passing all the props down to ExampleCoordinates to make things easier. I'll update its proptypes to make it more clear what is needed